### DISCUSSION TITLE
IR Code Database Usage

### DISCUSSION THREAD

**Discussion by PaulAnurag** [probonopd/irdb#101] [Q&A]
Hi,

Thank you for sharing the database of mostly all the brands. I am very new in this IR Remote learning and control. If I want to send data using any of this company's data, how can I do so?

Can anyone share me the steps.

Regards
Anurag

**Comment by probonopd** [comment]
Hello @PaulAnurag, check out http://www.harctoolbox.org/IrScrutinizer.html

**Reply by PaulAnurag** [reply]
Thank You for the response, will check it out.

Regards
Anurag
